src/bp/core/services/converse.ts

```diff
@@ -141,7 +141,8 @@ export class ConverseService {
 
     Object.assign(this._responseMap[event.target], <ResponseMap>{
       nlu: event.nlu || {},
-      state: _.get(event, 'state.temp', {})
+      state: event.state,
+      suggestedReplies: event.suggestedReplies
```

Comment:
Should we add the suggested replies to the response? Maybe only those that have a high enough confidence level? Just a thought.

Reply:
@rndlaine For debugging purposes I think we should add all of them because we want to see the confidence if low